src/cmdline/mod.rs

```diff
@@ -281,6 +281,16 @@ pub struct CommonCustomizeConfig {
     /// including when Ignition is run.
     #[clap(long, value_name = "path")]
     pub network_keyfile: Vec<String>,
+    /// Nmstate file to generate NetworkManager keyfile for live & dest
```

Comment:
It looks like it's typed "NMState".

Reply:
Official name is with just capital N, https://github.com/nmstate/nmstate/pull/224.

Reply:
Ahh interesting. Looks like the official website needs to be fixed then: https://nmstate.io/.

Reply:
Once CI is working for this PR, it'll reject this line as being too long.  You can check manually with `tests/help.sh`.

Reply:
Can we enforce this with `cargo fmt` ?

Reply:
The issue is that the short help text needs to be a single line of a certain maximum length, which varies depending on the width of the flags.  I'm not sure `cargo fmt` is configurable to that degree.

Reply:
Done